src/typing/errors/flow_error.ml

```diff
@@ -1021,6 +1021,7 @@ let rec make_error_printable (error : Loc.t t) : Loc.t Errors.printable_error =
     | (Some loc, Error_message.Normal { features }) ->
       mk_error ~trace_infos ~kind loc (code_of_error error) features
     | (None, UseOp { loc; features; use_op }) -> mk_use_op_error loc use_op features
+    | (Some _, UseOp { loc; features; use_op }) -> mk_use_op_error loc use_op features
```

Comment:
This is probably wrong.
I returned `UseOp` in error_message.ml for `EFunPredCustom`, but that other loc in the tuple is apparently still there...